hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelinePlacementPolicy.java

```diff
@@ -315,6 +314,50 @@ DatanodeDetails fallBackPickNodes(
     return results;
   }
 
+  private DatanodeDetails randomPick(List<DatanodeDetails> healthyNodes) {
```

Comment:
I think a Java doc would be useful for this method to explain how it works. It seems to pick two random nodes and then return the one with the less load - is that correct?

Reply:
Yea. Your understanding is correct. I will add doc as description.